.github/workflows/docker-build.yml

```diff
@@ -138,7 +138,7 @@ jobs:
         # Load the image to the Docker daemon if it is required as a base image for the Jib build.
         echo "LOAD_IMAGE=$([[ '${{ matrix.docker.task }}' != '' ]] && echo 'true' || echo 'false')" >> $GITHUB_ENV
         # Push the image to the registry if it is not a PR and no Jib build is configured.
-        echo "PUSH_IMAGE=$([[ ${{ env.IS_PR }} == 'false' && '${{ matrix.docker.task }}' == '' ]] && echo 'true' || echo 'false')" >> $GITHUB_ENV
+        echo "PUSH_IMAGE=$([[ ${{ env.IS_PR }} == 'false' && '${{ matrix.docker.task }}' != *':jib' ]] && echo 'true' || echo 'false')" >> $GITHUB_ENV
```

Comment:
I thinkt this will always be a jib task so you could also just remove the whole condition `&& '${{ matrix.docker.task }}' == ''`.

Reply:
Well, currently these are `jibDockerBuild` tasks.